src/daemon/rpmostree-sysroot-upgrader.cxx

```diff
@@ -73,6 +73,7 @@ struct RpmOstreeSysrootUpgrader {
   OstreeDeployment *cfg_merge_deployment;
   OstreeDeployment *origin_merge_deployment;
   RpmOstreeOrigin *origin;
+  RpmOstreeOrigin *computed_origin;
```

Comment:
I'd be happier if we can keep only a single `RpmOstreeOrigin *` in this scope (i.e. the computed one), see longer comment in the PR.